docs/API.md

```diff
@@ -0,0 +1,1360 @@
+# Frontend Platform API Documentation
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
+
+### Table of Contents
+
+-   [Platform APIs][1]
+    -   [Initialization][2]
+        -   [analytics][3]
+        -   [auth][4]
+        -   [history][5]
+        -   [initError][6]
+        -   [initialize][7]
+    -   [Config][8]
```

Comment:
Config, PubSub, and Init are all top level exports right? Does this doc reflect that?